contracts/src/v0.8/functions/tests/1_0_0/FunctionsCoordinator.t.sol

```diff
@@ -150,8 +152,47 @@ contract FunctionsBilling_OracleWithdraw {
 }
 
 /// @notice #oracleWithdrawAll
-contract FunctionsBilling_OracleWithdrawAll {
-
+contract FunctionsBilling_OracleWithdrawAll is FunctionsMultipleFulfillmentsSetup {
+  function setUp() public virtual override {
+    // Use no DON fee so that a transmitter has a balance of 0
+    s_donFee = 0;
+
+    FunctionsMultipleFulfillmentsSetup.setUp();
+  }
+
+  function test_OracleWithdrawAll_RevertIfNotOwner() public {
+    // Send as stranger
+    vm.stopPrank();
```

Comment:
Why do we need stopPrank() first? Aren't tests independent?

Reply:
The initial setup set the prank to the "owner" aka deployer of the contracts, so we stop that one first to take on the stranger role